base/src/main/java/info/metadude/kotlin/library/c3media/models/Metadata.kt

```diff
@@ -4,7 +4,7 @@ import com.squareup.moshi.Json
 
 data class Metadata(
 
-        val related: List<Int>? = null,
+//        val related: List<Int>? = null,
```

Comment:
Until the API discussion has a consensus this field could be changed to `val related: Map<String, Int>? = null,` which resolved the parsing issue.

Reply:
Cool. If you already figured out that this works, maybe you can add this one and I'll trim down this PR to only contain the new field?
Or I could also play around with this fix and add it in a second PR. But this is probably not going to happen before end of the week.

Reply:
☑️ I fixed parsing the `related` property [here](1e4a62fb42145d22bf11f2d11646ea0fc0f616fd) and merged it into `master`.